api/scripts/wazuh-apid.py

```diff
@@ -106,6 +108,7 @@ def start():
 
     # Enable cache plugin
     if api_conf['cache']['enabled']:
+        logger.warning(CACHE_DEPRECATED_MESSAGE.format(release="4.8"))
```

Comment:
```suggestion
        logger.warning(CACHE_DEPRECATED_MESSAGE.format(release="4.8.0"))
```

Reply:
Fixed in commit b3668e8263fe0eccdd4d6da019d3d55d4bcd170c.